examples/waspello/README.md

```diff
@@ -0,0 +1,26 @@
+Waspello 
+=========
+
+**Waspello** is a trello clone app made with Wasp.
+
+This app is deploted at [https://waspello.netlify.app/](https://waspello.netlify.app/).
+
+# Development
+
+### Databse
```

Comment:
If we are repeating this for every example app, do we already have a place in the docs where we could just link to?

Reply:
This is the place in the docs explaining how to run your postgres db, so we can just link to it: https://wasp-lang.dev/docs/language/features#postgresql

Reply:
Yup that would be the best to do...